src/components/Input/Input.js

```diff
@@ -14,83 +14,81 @@ import {
   InputRequiredStyled
 } from './InputStyled';
 
-class Input extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    const {
-      type,
-      placeholder,
-      value,
-      onChange,
-      onBlur,
-      error,
-      showVisibilityIcon,
-      handleClickShowPassword,
-      showPassword,
-      passwordStrength,
-      ariaRequired,
-      ariaInvalid,
-      icon,
-      required,
-      floatingLabels,
-      reference
-    } = this.props;
-
-    return (
-      <InputComponentStyled>
-        <InputElementWrapperStyled
-          error={error}
-          passwordStrength={passwordStrength}
+const Input = ({
+  type,
+  placeholder,
+  value,
+  onChange,
+  onBlur,
+  error,
+  showVisibilityIcon,
+  handleClickShowPassword,
+  showPassword,
+  passwordStrength,
+  ariaRequired,
+  ariaInvalid,
+  icon,
+  required,
+  floatingLabels,
+  reference
+}) => {
+  return (
+    <InputComponentStyled>
+      <InputElementWrapperStyled
+        error={error}
+        passwordStrength={passwordStrength}
+      >
+        {/* {icon && <InputIconStyled>{icon.render()}</InputIconStyled>} */}
```

Comment:
can this be deleted?

Reply:
I was thinking about that line but you're right. I will delete it